.github/workflows/model-evaluation.yml

```diff
@@ -0,0 +1,201 @@
+name: Full Model Evaluation
+
+on:
+  workflow_dispatch:
+    inputs:
+      provision-and-deploy:
+        description: 'Whether to provision the deployment'
+        required: true
+        default: true
+        type: boolean
+      deploy-only:
+        description: 'Whether to deploy the application'
+        required: true
+        default: false
+        type: boolean
+      purge:
+        description: 'Whether to purge the deployment after evaluation'
+        required: true
+        default: false
+        type: boolean
+
+permissions:
+  id-token: write
+  contents: read
```

Comment:
Why do we need these for this workflow?

Reply:
Because I'm using [Azure Login](https://github.com/Azure/login) workflow to login to azure. And this workflow requires these two permissions.

Reply:
Got it, cheers!